<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);
font: 9pt &quot;Comic Sans MS&quot;;
background-color: rgb(51, 0, 152);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Valeur en Euro</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_euros"/>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 170, 255);

font: 12pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Convertir</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Valeur en Dollar</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_dollar"/>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="label_message">
      <property name="text">
       <string>Taux de conversion: 1 euro = 1,05 dollar am√©ricain</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;Algerian&quot;;</string>
      </property>
      <property name="text">
       <string>Convertisseur</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>327</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionTaux_de_change"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuPropos">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionApropos"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuPropos"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionTaux_de_change">
   <property name="text">
    <string>Taux de change</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionApropos">
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
